.github/workflows/rebuild-clade-i.yaml

```diff
@@ -0,0 +1,60 @@
+name: Rebuild clade-I
+
+on:
+  repository_dispatch:
+    types:
+      - rebuild
+      - rebuild_clade-i
+
+  workflow_dispatch:
+    inputs:
+      trial_name:
+        description: "If set, result will be at nextstrain.org/staging/trial/${trial_name}/${auspice_name}"
+        required: false
+
+jobs:
+  set_config_overrides:
+    runs-on: ubuntu-latest
+    steps:
+      - id: config
+        name: Set config overrides
+        env:
+          TRIAL_NAME: ${{ inputs.trial_name }}
+        run: |
+          config=""
+          if [[ "$TRIAL_NAME" ]]; then
+            config+="--config"
+            config+=" deploy_url='s3://nextstrain-staging/'"
+            config+=" auspice_prefix='"$TRIAL_NAME"'"
+          fi
+
+          echo "config=$config" >> "$GITHUB_OUTPUT"
+    outputs:
+      config_overrides: ${{ steps.config.outputs.config }}
+
+  rebuild_clade_i:
+    needs: [set_config_overrides]
+    permissions:
+      id-token: write
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@master
+    secrets: inherit
+    with:
+      runtime: conda
```

Comment:
_non-blocking_

Just out of curiosity, why `conda` and not `docker`?

Reply:
Oh, I'm open to `docker` - does this work smoothly inside a GitHub action? I'm happy to switch to that.

Reply:
Yup! Most of the smaller pathogen builds, e.g. [zika](https://github.com/nextstrain/zika/blob/d588b41e2549c74c14105a0515f09c416e1e51a0/.github/workflows/phylogenetic.yaml#L86-L89), use `docker`.

Reply:
👍 [switched to docker](https://github.com/nextstrain/mpox/compare/243f6c5d7ccd80c1631161cfef12ff353cbe9a24..ec5aa8bb2fe7b56ade11a6f9d566a7955f8c587b)